webapp/frontend/.gitignore

```diff
@@ -33,3 +33,4 @@ yarn-error.log*
 # typescript
 *.tsbuildinfo
 next-env.d.ts
+package-lock.json
```

Comment:
add a newline at the end of the file to remove this issue

Reply:
Fixed